src/ppar.rs

```diff
@@ -4,6 +4,7 @@ use super::*;
 use crate::{Error, Result};
 
 /// Persistent array using rope-data-structure.
+#[derive(Debug)]
```

Comment:
So there is this method `pretty_print()` available on Vector and Node types. The primary intention was to be able to debug the tree while testing. I am not sure Debug will be of practical use when the Tree depth is large, and testing it with big data-set.

I am okay with this patch, just wanted to know the reason for adding this. Thanks,